packages/react-md/rollup.config.js

```diff
@@ -3,43 +3,67 @@ import commonjs from '@rollup/plugin-commonjs';
 import replace from '@rollup/plugin-replace';
 import { terser } from 'rollup-plugin-terser';
 
-module.exports = {
-  input: 'src/index.ts',
-  output: [
-    {
-      file: 'dist/umd/react-md.development.js',
-      name: 'ReactMD',
-      format: 'umd',
-      globals: {
-        react: 'React',
-        'react-dom': 'ReactDOM',
+function createConfig(type) {
+  const globals = {
```

Comment:
Doesn't look like the `globals` and `external` correctly separate `react-md` from the icon bundles. I think I'll rename them to:

- react-md-svg-icons.development.js
- react-md-svg-icons.production.min.js
- react-md-font-icons.development.js
- react-md-font-icons.production.min.js

and keep all the bundle names the same as `ReactMD`. I'll update this tomorrow.